### DISCUSSION TITLE
Performance MS SQL connector

### DISCUSSION THREAD

**Discussion by JeroenvdBurg** [Budibase/budibase#6273] [Help]
Hi!

First of all the budibase product is great, you can create CRUD interfaces with in minutes and it looks amazing.

 One thing which causes some problems for us in using it is the performance with MS SQL. I have created a connection with an Azure SQL database. When creating a simple table view data it takes 5 sec to fetch the data. When looking at the chrome network tab is it is the search API call that takes so long.  

It is not caused by the Azure SQL db performance because when using appsmith it loads the data instantly. 

Anyone else experiencing this behavior and/or have a solution for this ? 

![image](https://user-images.githubusercontent.com/30255073/173123177-38c2f0b7-ffd8-4b17-92df-39cda43c9d66.png)

**Comment by melohagan** [comment]
Hi @JeroenvdBurg 

Thank you for trying out Budibase!

How much data are you pulling back? 
Is it creating a table within the designer and assigning a data provider, or within the data tab itself that you're seeing the issue?
Do you have any filters?

**Reply by JeroenvdBurg** [reply]
Hi! 

thanks for your reply. It is a very basic table with only 3 columns and 8 rows. It is both in the data tab as in the app with a datatable. No filters applied. 

If it helps I can share my projectId. 

BTW I have copied the same table to a PostgreSQL database and it much faster in budibase.
